content/bn/docs/reference/glossary/controller.md

```diff
@@ -0,0 +1,29 @@
+---
+title: কন্ট্রোলার
+id: কন্ট্রোলার
```

Comment:
we don't need to change the value of `id`.
you can keep it as before. `id: controller`